rls/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id "java"
+    id "java-library"
```

Comment:
This will not do anything more than `java` plugin unless you use `api` dependency. grpc-lrs is like grpc-grpc-lb and grpc-xds, they don't provide public APIs for users. So `java-library` is not necessary.

Reply:
ah okay that makes sense.